.circleci/config.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       JVM_OPTS: -Xmx3200m
       TERM: dumb
       # LaScala Circle CI latest successful build. Can be found from: https://circleci.com/gh/rchillyard/LaScala/tree/master
-      LaScalaBuildNumber: 10
+      LaScalaBuildNumber: 61
```

Comment:
It seems CircleCI API now supports fetching the latest build artifacts 
See API docs [Artifacts of the latest Build](https://circleci.com/docs/api/v1-reference/#build-artifacts-latest)
I think this will help to avoid maintaining the build number.
That will require changing the `$LaScalaBuildNumber` in the curl command in line 40 with the word `latest`

Reply:
Just noticed, the above will not work if the latest build is for a different version of LaScala than what's used in this project

Reply:
@Benaiad I totally agree with you. I was intending to use [Project API](https://circleci.com/docs/api/v1-reference/#projects) to get latest build. But your approch of using Artifact of latest build looks more neet. I will work on this tomorrow.